openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts

```diff
@@ -13,9 +13,9 @@
 
 /* eslint-disable max-len */
 
-export const LATEST_VERSION_ID = 9;
+export const LATEST_VERSION_ID = 10;
 
-export const COOKIE_VERSION = 'VERSION_0_12_1'; // To be changed with each release.
+export const COOKIE_VERSION = 'VERSION_0_13_0'; // To be changed with each release.
```

Comment:
Exactly where we are using this?

Reply:
We are using this in 2 places,
1. to keep track of the recently viewed and recently searched data for the current version
2. for what's new modal, if a user is upgrading from the old version to the new version then on the first login we show the modal based on this cookie.